examples/with-scss/razzle.config.js

```diff
@@ -0,0 +1,73 @@
+// Based on: https://gist.github.com/jaredpalmer/0a91a7bd354b875b913c74f4b16125f7
+
+const path = require('path')
+const autoprefixer = require('autoprefixer')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+
+module.exports = {
+  modify: (baseConfig, { target, dev }, webpack) => {
+
+    const appConfig = Object.assign({}, baseConfig)
+    const isServer = target !== 'web'
+
+    const postCssLoader = {
+      loader: 'postcss-loader',
+      options: {
+        ident: 'postcss', // https://webpack.js.org/guides/migrating/#complex-options
+        plugins: () => [
+          autoprefixer({
+            browsers: [
+              '>1%',
+              'last 4 versions',
+              'Firefox ESR',
+              'not ie < 9' // React doesn't support IE8 anyway
+            ]
+          })
+        ]
+      }
+    }
+
+    appConfig.module.rules.push({
+      test: /.scss$/,
+      use:
+      // On the server, we simply use css-loader to deal with scss imports
+      isServer ? 'css-loader' :
+      // For development, include source map
+      dev
+      ? [
+        'style-loader',
+        {
+          loader: 'css-loader',
+          options: {
+            modules: false,
+            sourceMap: true,
+          },
+        },
+        postCssLoader,
+        'sass-loader',
+      ]
+      // For production, extract CSS
+      : ExtractTextPlugin.extract({
+        fallback: 'style-loader',
+        use: [
+          {
+            loader: 'css-loader',
+            options: {
+              importLoaders: 1,
```

Comment:
This means that when we `@import` inside an SCSS file, only `post-css-loader` will run on it and not `sass-loader` - is this the correct expectation? I have been trying to understand the implications. I understand that the original gist by @jaredpalmer also has this - but we could add a comment here about what does this actually do.

Reply:
I copied this sass setup from nytimes/kyt should look at their latest release and copy.

Reply:
I read up more on this - what we have here is correct. It basically tells `css-loader` to use `postcss-loader` on all `@import` lines that it comes across in CSS.